showcase/app/templates/components/modal.hbs

```diff
@@ -239,6 +239,42 @@
     </Hds::Modal>
   {{/if}}
 
+  <button type="button" {{on "click" (fn this.activateModal "longModalActive")}}>Open long content modal</button>
+
+  {{#if this.longModalActive}}
```

Comment:
I believe we already have this coming in https://github.com/hashicorp/design-system/commit/82b541686967c11f257d46d904dcfb83477a41b2, but we can drop it or rely on merge

Reply:
Yes, that was the intent. I cherry-picked the commit. Hopefully it will not cause conflicts (and even if it does, it should be an easy fix)